src/pages/Home.js

```diff
@@ -83,26 +83,37 @@ function Header({ carePlan, patient }) {
   );
 }
 
-const HomeContent = ({ carePlan }) => {
-  const tabs = carePlan
-    ? [
-        {
+const getCarePlanTabs = (carePlan) => {
+  const tabOrder = process.env.REACT_APP_HOME_TAB_ORDER ?
+    process.env.REACT_APP_HOME_TAB_ORDER.split(',').map((tabName) => tabName.toLowerCase()) :
+    ["tasks", "goals", "detais"]
+  return tabOrder.map((tabName) => {
+    switch (tabName) {
+      case "tasks":
+        return {
           title: "Tasks",
-          content: <TaskCards carePlan={carePlan} />,
+          content: <TaskCards carePlan={carePlan}/>,
           to: `/home/${carePlan.id}/tasks`,
```

Comment:
And reuse the variable `to` here and in the following objects
```suggestion
          to,
```